libsoftwarecontainer/src/gateway/gateway.cpp

```diff
@@ -126,13 +126,13 @@ bool Gateway::isActivated()
     return m_state >= GatewayState::ACTIVATED;
 }
 
-ReturnCode Gateway::setEnvironmentVariable(const std::string &variable, const std::string &value)
+bool Gateway::setEnvironmentVariable(const std::string &variable, const std::string &value)
 {
     if (hasContainer()) {
-        return getContainer()->setEnvironmentVariable(variable, value);
+        return isSuccess(getContainer()->setEnvironmentVariable(variable, value));
```

Comment:
Yes, but to avoid making the commit too big (I had already changed in ~20 files) I had to stop somewhere. Is this okay with you, @tobsan @breakreturn?